ls1intum/Artemis
# Remove unused rest call

### Checklist

### Motivation and Context



While i was working on my integration tests i found this rest route, which existed on the client, but not on the server. After investigating the test-repository functionality we decided to remove the rest call, since the test-repository does not have build logs.

### Steps for Testing
There are no steps for testing the "new" functionality, but in order to ensure that remaining functionality is not effected by the change please use the following steps 

1. Create a Programming Exercise with an enabled Online Editor
2. Open the editor and create a compile error (for example remove curly brackets)
3. Click on the submit button
=> the build should fail and a build output should be shown
